assets/src/edit-story/components/header/buttons/index.js

```diff
@@ -41,21 +41,32 @@ const ButtonList = styled.nav`
 
 const List = styled.div`
   display: flex;
+  align-items: center;
 `;
 
 const Space = styled.div`
   width: 6px;
```

Comment:
Hm, I can actually see only 12px currently, could you share a link where the 8px can be seen? (Although this might become irrelevant since the design is not fully confirmed yet.)

Reply:
It happens on the smaller breakpoints:
![Screen Shot 2021-02-17 at 01 41 03](https://user-images.githubusercontent.com/637548/108165849-4004fd80-70c1-11eb-9c35-ae664e1f9482.png)

Reply:
(and the [link](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=327%3A10229) you asked for)